src/api/buildSwapTransaction.ts

```diff
@@ -39,8 +39,14 @@ export async function buildSwapTransaction(
     };
   }
   if (params.maxSlippage) {
+    let slippagePercentage = params.maxSlippage;
+    // Adjust slippage for V1 API (aggregator)
+    // V1 expects slippage in tenths of a percent (e.g., 30 = 3%)
+    if (params.useAggregator) {
```

Comment:
I still don't follow, is `useAggregator: true` what everyone uses at the moment?

If that so, we should not change this code.

Reply:
There are two different APIs, V1 [wallet/swap-service](https://github.cbhq.net/cloud/swap-router) and V2 [cloud/swap-router](https://github.cbhq.net/cloud/swap-router). 

V1:
- `useAggregator: true`
- Expects slippage in tenths of a percent (e.g., 30 = 3%) which is why we must multiply maxSlippage by 10 before sending the request. 

V2:
- `useAggregator: false`
- Expects slippage in a percent, which is why we do not have to adjust the maxSlippage value.

PartyDOA is the only known customer who is using the maxSlippage parameter.